src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -1,11 +1,18 @@
 package com.wikia.webdriver.elements.common;
 
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
 
+import org.openqa.selenium.TimeoutException;
+
 public class Navigate extends BasePageObject {
 
   public Navigate toPage(String pageName) {
-    driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
+    try {
+      driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
+    }catch (TimeoutException e){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)